### PR TITLE
LPS-139003 Plus icon to add an account disappears when selecting an account the admin has created

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

File: portal-impl/src/resource-actions/portal.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 		<portlet-name>90</portlet-name>
 		<permissions>
 			<supports>
+				<action-key>ADD_ACCOUNT_ENTRY</action-key>
```
Comment (drewbrokke) [inline portal-impl/src/resource-actions/portal.xml:9]: I think we can still keep this in the `account-service/src/main/resources/resource-actions/default.xml` file, kind of like what we do for the Organization actions (e.g. `MANAGE_ACCOUNTS`).

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just one comment, @pei-jung. Everything else looks good!

File: portal-impl/src/resource-actions/portal.xml (hunk 1 above)
Comment (pei-jung) [reply portal-impl/src/resource-actions/portal.xml:9]: I've tried that, but the action won't be added. Looks like portal 90 is initialized after the modules are registered.

Comment (drewbrokke) [reply portal-impl/src/resource-actions/portal.xml:9]: Ok, makes sense. We're good to go then 👍

**Comment by drewbrokke** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:test:upgrade

**Comment by patriciajeanperez** [comment]
ci:test:upgrade

**Comment by pei-jung** [comment]
@patriciajeanperez are the failures in the upgrade suite related to the changes in this PR?

**Comment by patriciajeanperez** [comment]
hey @pei-jung, sorry for the delayed response. I didn't see any related upgrade failures

**Comment by pei-jung** [comment]
@patriciajeanperez thanks for checking. So is this safe to forward?

**Comment by pei-jung** [comment]
@drewbrokke fixed rebase error and force-pushed

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Good morning, @pei-jung! Found these [failures](https://testray.liferay.com/reports/production/logs/2021-09/test-1-30/test-portal-acceptance-pullrequest(master)/605/modules-integration-mysql57-jdk8/0/1/jenkins-console.txt.gz) in relevant. 

` java.lang.AssertionError: Thread Thread[http-nio-8080-exec-6,5,main] caught concurrent failure: java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.vulcan.internal.jaxrs.exception.mapper.ExceptionMapper, message=java.lang.NullPointerException
     [exec]     java.lang.AssertionError: {level=ERROR, loggerName=com.liferay.portal.vulcan.internal.jaxrs.exception.mapper.ExceptionMapper, message=java.lang.NullPointerException`

`Caused by: java.lang.NullPointerException
     [exec]     	at com.liferay.account.internal.security.permission.resource.AccountEntryModelResourcePermission.contains(AccountEntryModelResourcePermission.java:94)
     [exec]     	at com.liferay.portal.vulcan.util.ActionUtil._addAction(ActionUtil.java:185)
     [exec]     	at com.liferay.portal.vulcan.util.ActionUtil.addAction(ActionUtil.java:108)
     [exec]     	at com.liferay.headless.admin.user.internal.resource.v1_0.BaseAccountResourceImpl.addAction(BaseAccountResourceImpl.java:1097)
     [exec]     	at com.liferay.headless.admin.user.internal.resource.v1_0.AccountResourceImpl.getAccountsPage(AccountResourceImpl.java:145)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179)
     [exec]     	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)
     [exec]     	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:201)
     [exec]     	... 94 more`

**Comment by pei-jung** [comment]
Fixed test failure and resent as https://github.com/liferay-usm/liferay-portal/pull/280

**Pull request closed without merging**